pkg/system/path_windows.go

```diff
@@ -2,6 +2,26 @@
 
 package system
 
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
 // DefaultPathEnv is deliberately empty on Windows as the default path will be set by
 // the container. Docker has no context of what the default path should be.
 const DefaultPathEnv = ""
+
+// CheckSystemDriveAndRemoveDriveLetter verifies that a path, if it includes a drive letter,
+// is the system drive.
+func CheckSystemDriveAndRemoveDriveLetter(path string) (string, error) {
+	if filepath.IsAbs(path) {
+		if strings.ToUpper(string(path[0])) != "C" {
```

Comment:
Good to know about EqualFold, wasn't aware of it. In this context, I really only need to verify the first character.